ls1intum/Artemis
# [Bugfixes] Multiple alignment/button bugfixes

### Checklist
- [x] I tested all changes

### Motivation and Context


Some buttons in Artemis are not aligned correctly.
Other not necessary buttons are removed.
Also the "Correction Round 1" Column is changed to "Assessment Action" in all assessment dashboards.

### Description
1. Center align of the Test Run button:
 <img src="https://user-images.githubusercontent.com/33342534/107973672-cc29fe80-6fb5-11eb-91d9-655ca4022551.png" width="200"> -->  <img src="https://user-images.githubusercontent.com/33342534/107987247-f5568900-6fcd-11eb-975e-08da3ebfc8d2.png"  width="200">

2. Replace "Correction Round 1" with "Assessment actions" When there is only one round.
<img src="https://user-images.githubusercontent.com/33342534/107987436-4797aa00-6fce-11eb-9cc3-b5479e9af8a8.png" width="200"> --> <img src="https://user-images.githubusercontent.com/33342534/107988116-9e51b380-6fcf-11eb-9499-feb9374632e3.png" width="185"> 

3. Change Buttons in the exercise overview: remove "View" Button for all types as you can simply click on the id/title of the exercise. Also modified programming exercise buttons to look cleaner.
![image](https://user-images.githubusercontent.com/33342534/108036832-0688af80-7039-11eb-9690-76164c9f7f02.png)
![image](https://user-images.githubusercontent.com/33342534/107990890-bfb59e00-6fd5-11eb-959b-4173389c3051.png)

4. Changed the "Test run ribbon".
<img src="https://user-images.githubusercontent.com/33342534/108116806-6a908f80-709c-11eb-82c7-4768afdab582.png" width="200"> -->  <img src="https://user-images.githubusercontent.com/33342534/108116717-4b91fd80-709c-11eb-8e17-eba44238a97a.png" width="200">

### Steps for Testing
1. Go to the exam overview of a course -> Test Runs -> see if the "Assess your testrun" button is vertically centered
2. Check in the [exercise-type]-exercise-assessment-dashboard that the last column(s) are correct according to the number of correctionrounds.
3. Check the buttons in the course -> exercise overview.
4.  Go to an exam and check the test run ribbon, and the test run summary ribbons.
